tests/consensus_tests/test_routing_token.py

```
@@ -0,0 +1,158 @@
+    r = upsert_points(
+        peer_uri,
+        [{"id": POINT_ID, "vector": VECTOR, "payload": {"marker": marker}}],
+        wait="true",
+        ordering="strong",
+    )
+    assert_http_ok(r)
+
+
+def retrieve_marker(peer_uri, headers={}):
+    """Retrieve the test point and return its `marker`, routing with the given headers."""
+    r = requests.post(
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)